bin/filter/filter_service_std.py

```diff
@@ -43,22 +43,22 @@ def main():
         den = int(row['total_volume']) if row['total_volume'] else 0
 
         # performance = volume_meeting_target / total_volume
-        if num <= 0 or den <= 0:
+        if den <= 0:  # NaN calculation
             row['performance'] = None
         else:
             row['performance'] = max(round(num / den, 4), 0)
 
         # calculate target_met
-        if row['target']:
+        if row['target'] is not None:  # target can be 0
             target = float(row['target'])
 
-            # if no performance(volume_meeting_target/total_volume)
-            # then target_met is not possible
-            if row['performance'] is None:
+            if row['volume_meeting_target'] is None or row['total_volume'] is None:
```

Comment:
If you want `row['target_met'] = 'NA'` when row['total_volume'] is <= 0 then you could instead
```suggestion
            if row['volume_meeting_target'] is None or row['total_volume'] is None or row['performance'] is None:
```

Reply:
hmmm yeah I think that would make sense. As we could not calculate the performance if the `total_volume` is zero, thus the `target_met` should be `NA`

Reply:
I don't know anymore. this logic is getting silly and they agreed on it originally. like `N` vs `NA` ugh